docs/deploy.md

```diff
@@ -4,4 +4,8 @@ If you would like to run your own instance of this app, see the [docs for deploy
 
 This app requires these **Permissions & events** for the GitHub App:
 
-> **TODO**: List permissions required for deployment here. See [probot/stale](https://github.com/probot/stale/blob/master/docs/deploy.md) for an example.
+- Pull requests - **Read & Write**
+  - [x] Check the box for **Pull request** events
+- Repository webhooks - **Read-only**
+- Commit statuses - **Read & Write**
+  - [x] Check the box for **Status** events
```

Comment:
I think permissions for https://github.com/uber-workflow/probot-app-pr-label/blob/848ad966c2691d319226f7adff669e896e253cae/src/index.js#L19 might be needed (just like the stale bot)

Reply:
Fair point!  I'll update this to include:

```
- Single File - **Read-only**
   - Path: `required-labels.yml`
```

Reply:
Maybe not obvious, but this method implicitly looks in`.github` https://probot.github.io/api/latest/classes/context.html#config. I've updated the path accordingly

Reply:
Womp, womp.  Thank you!